skbeam/io/tests/test_fit2d_save.py

```diff
@@ -44,17 +44,24 @@
 import numpy as np
 from skbeam.io.fit2d import fit2d_save, read_fit2d_msk
 from numpy.testing import assert_array_equal
-import pytest
+import sys
+
+if sys.version_info >= (3, 0):
+    from tempfile import TemporaryDirectory
+else:
+    from backports.tempfile import TemporaryDirectory
+
 
-@pytest.mark.xfail(raises=IOError)
 def test_save_output_fit2d():
```

Comment:
I doubt that @danielballan led you astray 🤷‍♂️ . He always provides excellent advice AFAICT. 

You can manually clean up if you want by throwing in a `tmpdir.remove(t)` at the end of the function, but I'd **highly** recommend leaving the test data in there. It is generally a very useful feature to be able to inspect whatever is being written by these test functions in case of error. 

Also pytest keeps like 5-7 iterations but automatically deletes stuff older than that. I doubt that this test suite is writing enough data where 5-7 iterations is going to cause problems.  

xref: https://github.com/pytest-dev/pytest/issues/543